src/elm/Route/Query.elm

```diff
@@ -11,6 +11,8 @@ import Url.Builder exposing (QueryParameter)
 import Url.Parser exposing (query)
```

Comment:
i think this whole file will get overwritten by `elm-land generate`, should we remove these comments?

Reply:
comments removed.